app/dashboard/templates/bounty/new.html

```diff
@@ -22,6 +22,7 @@
   {% include 'shared/head.html' %}
   {% include 'shared/cards.html' %}
   <link rel="stylesheet" href={% static "v2/css/submit_bounty.css" %}>
+  <link rel="stylesheet" href="{% static "v2/css/jquery.select2.min.css" %}">
```

Comment:
This template already `includes` shared/head.html which already has `v2/css/jquery.select2.min.css` in it.  This can be removed.